src/vs/platform/windows/electron-main/windows.ts

```diff
@@ -4,21 +4,21 @@
  *--------------------------------------------------------------------------------------------*/
 
 import electron from 'electron';
+import { Color } from 'vs/base/common/color';
```

Comment:
@justschen here again, I wonder how all the imports changed?

Reply:
ooh yes this was just with `editor.codeActionsOnSave` and `organizeImports`

should i revert the instances where this happens?

Reply:
Its fine for now but for future PRs in files you did not author I would not commit the changes.